ruby-challenges.rb

```diff
@@ -3,32 +3,58 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
-reposts1 = 7
-# Expected output: '7 is odd'
-reposts2 = 42
-# Expected output: '42 is even'
-reposts3 = 221
-# Expected output: '221 is odd'
+    # Pseudocode:
+        # Create Ruby method named 'even_odd'
+        # Use the '.even?' method that checks to see if given number is even. If given number is even, method will return a string that says "is even."
+        # Use the '.odd?' method that checks to see if a given number is odd. If given number is odd, method will return a string that says "is odd."
+        # Output will be the given number using string interpolation with saying "#{number} is even" or "#{number is odd}"
+
+            def even_odd(number)
+                if number.even? 
+                    "#{number} is even"
+                else number.odd? 
+                    "#{number} is odd"
+                end
+            end
+
```

Comment:
Don't need to give `else` a condition as it is a catch all for when something does not meet the condition set in the `if` statement.